dot/state/grandpa_test.go

```diff
@@ -220,8 +220,6 @@ func TestAddScheduledChangesKeepTheRightForkTree(t *testing.T) {
 	for tname, tt := range tests {
 		tt := tt
 		t.Run(tname, func(t *testing.T) {
-			t.Parallel()
```

Comment:
why?

Reply:
As mentioned in a comment, we need to clear the `scheduledChangeRoots` after the test ends. So we can't run in parallel.

https://github.com/ChainSafe/gossamer/blob/bdd005337f743681b103c8e09c19b20d2555bc61/dot/state/grandpa_test.go#L222-L229